DESCRIPTION

```diff
@@ -52,7 +52,7 @@ Imports:
     coda,
     distr6,
     dplyr,
-    dust (>= 0.11.6),
+    dust (>= 0.11.18),
```

Comment:
This and mcstate being bumped here may cause problems as these versions are compatible with sircovid 0.12.12, but fits this week have not been run with that, and any downstream tasks won't be either

Reply:
Does this affect your task too Ed? One option is to create a version of sircovid that is compatible with this version of dust/mcstate but is the old sircovid

Reply:
It's fine for my PR - boosting before waning doesn't actually require any spimalot changes and latest version of sircovid (which is the one I need) requires the latest versions of dust/mcstate

Reply:
I would assume this should be safe, as after submitting this week's SPI-M ask we should all migrate to new package versions.

Reply:
The reason why the dust boost was needed was mcstate now assumes existance of the `rng_algorithm` method (this was the issue that Marc hit, which I incorrectly thought was the removal of deprecated methods), so there is some coordination required I think. If the fits don't work with the new sircovid we should probably just run some new ones, but I can also help get a temporary version of sircovid up and running if that proves hard.

Reply:
Basically if Divya doesn't upgrade until after everything is run for this week, it should be fine. But there'd be issues if e.g. we need to make further changes to spimalot for the MTPs

Reply:
Hopefully that shouldn't be required, as I left them a spimalot version that works well with old sircovid/mcstate/dust for running the MTPs. I'll just give them a heads up to avoid any upgrading until we get word from @MJomaba everything has been submitted

Reply:
thanks @pabloperguz